WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -76,6 +78,10 @@ public void Initialize(NodesCollection nodes, WasabiSynchronizer synchronizer, U
 			HashChain = synchronizer.BitcoinStore.HashChain;
 			UseTor = Global.Config.UseTor.Value; // Don't make it dynamic, because if you change this config settings only next time will it activate.
 
+			this.WhenAnyValue(x => x.StatusBacking)
+				.ObserveOn(RxApp.MainThreadScheduler)
```

Comment:
IMO with the removal of `...` animation this can be simplified more. I would try to remove the unnecessary complexity and try the `SubscribeOn` instead of `ObserveOn`. According to this description it fits:
http://reactivex.io/documentation/operators/observeon.html

Reply:
Reading about it, it's another programmer holy war. I'm not sure it's a good idea to get into that now. Note this topic to the next dev meeting and explain it to us.